Lib/test/test_httpservers.py

```diff
@@ -444,6 +444,45 @@ def test_head(self):
         self.assertEqual(response.getheader('content-type'),
                          'application/octet-stream')
 
+    def test_browser_cache(self):
```

Comment:
Done in commit 42edfe3856e0ae683dbffe3f7a75b9daec0898e8.
The datetime of last modification is stored in method setUp and used in the browser cache tests. I also added a test for the Last-Modified header.